### DISCUSSION TITLE
Is there any documentation on what predicates are implemented?

### DISCUSSION THREAD

**Discussion by mainrs** [tree-sitter/tree-sitter#1931] [Q&A] [answered]
I am specifically looking for the Rust implementation.

**Comment by ahlinc** [comment, marked as answer]
@mainrs I'm afraid that such documentation still doesn't exist yet, the full list of [predicates](https://tree-sitter.github.io/tree-sitter/using-parsers#predicates) implemented for the Rust binding you can lookup in the following piece of code: https://github.com/tree-sitter/tree-sitter/blob/fb5fbdd78700b20754da5e6b4f2b4317a912e2d2/lib/binding_rust/lib.rs#L1508-L1609
